src/main/java/frc/robot/util/Dashboards.java

```diff
@@ -27,6 +31,26 @@ public static void initVoltageDashboard() {
         }
     }
 
+    public static void initCurrentDashboard(){
+        if (currentNeedsInit){
+            ShuffleboardTab tab = Shuffleboard.getTab("Current Info");
+            //Arm Motors
+            tab.addDouble("Right Arm Motor Current",() -> PDH.getCurrent(18));
```

Comment:
All these PDH channels used in the `getCurrent` calls should be defined in Constants (maybe make a PDH class in constants to store them)